File: igardenconnectweb/model/PlantVM.ts
export class PlantVM {
    idPlant?: number;
    name?: string;
    species?: string;
    optimalTemperature?: number;
    soilMoisture?: number;
    airMoisture?: number;
    light?: number;

    
  constructor(
    idPlant?: number,
    name?: string,
    species?: string,
    optimalTemperature?: number,
    soilMoisture?: number,
    airMoisture?: number,
    light?: number,
  ) {
    this.idPlant = idPlant;
    this.name = name;
    this.species = species;
    this.optimalTemperature = optimalTemperature;
    this.soilMoisture = soilMoisture;
    this.airMoisture = airMoisture;
    this.light = light;
  }
}

File: iGardenConnectAPI/VM/PlantVM.cs
namespace VM
{
    public class PlantVM
    {
        public int IdPlant { get; set; }
        public string Name { get; set; }
        public string Species { get; set; }
        public DateTime? WateringInterval { get; set; }
        public double? OptimalTemperature { get; set; }
        public double? SoilMoisture { get; set; }
        public double? AirMoisture { get; set; }
        public double? Light { get; set; }

    }
}